rfc-142-split-database-instances.md

```diff
@@ -0,0 +1,50 @@
+# One database per RDS instance
+
+## Summary
+
+We currently use a one-instance-many-databases approach to hosting MySQL, Postgres, Mongo and Redis. These mega-databases make it difficult to perform major version upgrades – they need to be coordinated across all applications at the same time, increase the blast-radius of problems in a single database, and make it harder to appropriately size and monitor resources for an application.
+
+This RFC proposes that we split the single RDS instances for MySQL and Postgres, moving to a single RDS instance per database. It also proposes we apply the same principle to our self-hosted Mongo and Redis installations.
+
+This principle _could_ be applied to RabbitMQ, S3, etc., but it's out of scope for this RFC.
```

Comment:
```suggestion
This RFC proposes that when applications depend on a supportive service that is not used by other applications, such as a database, the supportive service is distinct from those used by other applications. It is applied particularly to RDS instances, but the principle can apply to other supporting processes such as Redis, Elasticsearch, S3.
```

A suggestion here to try make this a bit less database specific and more of an all-rounder principle.